src/lib/siema-react.tsx

```diff
@@ -40,6 +40,7 @@ class Siema extends React.PureComponent<SiemaReactProps> {
     public prev: SiemaBase['prev'] = (...args) => { this.siemaInstance.prev(...args); };
     public next: SiemaBase['next'] = (...args) => { this.siemaInstance.next(...args); };
     public goTo: SiemaBase['goTo'] = (...args) => { (this.siemaInstance.goTo as any)(...args); }; // TODO: improve types
+    public resizeHandler = () => { (this.siemaInstance as any).resizeHandler(); }; // TODO: also improve types, or change siema to smth else
```

Comment:
`resizeHandler` jest raczej przewidziany do wewnętrznego użycia w Siemie aczkolwiek sam z siebie reaguje tylko na resize okna. Domyślam się, że chodzi o to, żeby Siema poprawiła sobie wymiary jeśli rozmiar okna się nie zmienił, ale zmieniliśmy zewnętrznie na przykład div, w który siema jest zapakowana. Jeśli tak to wypychałbym to z wnętrza "siema-react" pod inną, bardziej obrazującą sedno sprawy, nazwą.

Reply:
zmienione